docs/central-entities.rst

```diff
@@ -270,6 +373,12 @@ Similar to the Submission data detail page, the Entity detail page has some basi
 
 You can see the actual data stored in the Entity in the bottom left under the :guilabel:`Entity Data` section, and you can edit that information by clicking on :guilabel:`Edit` right there.
 
+(ADD IMAGE)
+
+When there is a conflict warning active on the Entity, you will see a notice above the Activity Feed. Here you can see a summary table of all the Entity updates that might be conflicting with each other. In the feed below, updates that are possibly in conflict will be highlighted. If you want a broad look at the updates in concern, the table is a great summary, but sometimes it can help to look at the changes described in each update in the Feed to really see what's going on.
+
+You can dismiss the conflict warning by pressing :guilabel:`Mark as Resolved` underneath the summary table. When you do this, the warning is dismissed. All values about this Entity currently in Central are now considered non-conflicting.
```

Comment:
Do you want to maybe explicitly mention that if there are issues you can use the edit button to make changes?